pi-prebuild.sh

```diff
@@ -32,6 +32,12 @@ apt-get install -y \
     libopencv-photo-dev \
     python3-dev
 
+echo "Install some useful tools for debugging"
+apt-get install -y \
+    screen \
+    htop \
+    vim
```

Comment:
`vim`? I thought you said these would be useful tools! ;p

Would `ipython` be worth including?